Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Use an official Python runtime as the parent image
+FROM python:3.8-slim-buster
```

Comment:
Just noticed this is on python 3.8 rather than 3.11.4

Please upgrade to the latest.